pkg/scheduler/util/utils.go

```diff
@@ -140,3 +140,227 @@ func MoreImportantPod(pod1, pod2 interface{}) bool {
 	}
 	return GetPodStartTime(pod1.(*v1.Pod)).Before(GetPodStartTime(pod2.(*v1.Pod)))
 }
+
+// Utilities below to build API objects using a "chained" manner
```

Comment:
ditto, where are those wrappers being used? I couldn't find any reference to them in your PRs (I think).

Reply:
Answered in https://github.com/kubernetes/kubernetes/pull/77760#discussion_r300890997